### PR TITLE
Testing - change output folder image writes, data dumps, performance result writes to CWD

### PR DESCRIPTION
**Pull request by r-abishek**
@kiritigowda This is a minor PR that changes output writes from testing to CWD.

### CODE REVIEW

File: utilities/test_suite/HIP/runTests.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 inFilePath2 = scriptPath + "/../TEST_IMAGES/three_images_mixed_src2"
 ricapInFilePath = scriptPath + "/../TEST_IMAGES/three_images_150x150_src1"
 qaInputFile = scriptPath + "/../TEST_IMAGES/three_images_mixed_src1"
+outFolderPath = scriptPath + "/../../../build"
```
Comment (r-abishek) [inline utilities/test_suite/HIP/runTests.py:36]: @kiritigowda The only assumption I make here is that the folder name is always "build" directly under "rpp". Is that okay?

Comment (r-abishek) [reply utilities/test_suite/HIP/runTests.py]: Fixed. Changed from build folder to CWD.

**Comment by kiritigowda** [comment]
@r-abishek need to use std filesystem for get_current_dir_name as its supported everywhere.

#include <filesystem>
#define get_current_dir_name std::filesystem::current_path

**Review by kiritigowda — APPROVED** [review]

**Comment by kiritigowda** [comment]
Tensor_host/hip - needs to be built in current path

**Comment by r-abishek** [comment]
@kiritigowda Could you merge this one?
Issuing a separate PR for moving the build folder of Tensor_hip/host to CWD. Don't want to mix with output folder dump changes

**Pull request merged**